### ISSUE TITLE
A couple of issues with the examples

### ISSUE THREAD

**Issue by andreww** [PFPE/shipgrav#18]
If, after installing everything, I run the example `dgs_bgm_comp.py` I get the following output:

![Figure_1](https://github.com/user-attachments/assets/1fa5e549-9d84-42d5-8a00-526aeb326629)

which is not what I expected from looking in the documentation. I've not looked in any detail but wonder if there is something not quite right with the data download for this example?

For `dgs_raw_comp.py` I get the error:

```
python dgs_raw_comp.py   
cleaning serial files: 100%|███████████████████████| 5/5 [00:00<00:00, 17360.53it/s]
Traceback (most recent call last):
  File "/Users/andreww/Code/shipgrav/shipgrav/example-scripts/dgs_raw_comp.py", line 157, in <module>
    meter_conf = tm.load(open(opath, 'rb'))
                         ~~~~^^^^^^^^^^^^^
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

which may be as simple as a typo.

All the other examples run fine and give the output shown in the documentation. 

(This is part of the JOSS review at https://github.com/openjournals/joss-reviews/issues/7358)

**Comment by hfmark** [comment]
That's odd - I can't reproduce either of those errors, even with deleting my pooch cache and re-downloading the data. It's probably worthwhile to try re-pip-installing the new version I just pushed to PyPi and making sure you have the latest example script files pulled from github (though I'm guessing you've done that already), and if that doesn't help, any further info you can dig out would be much appreciated for debugging purposes!

**Comment by hfmark** [comment]
Ah, I thought that might be it. I think I found the bug - it's because I didn't explicitly time-sort the BGM data, and I guess different systems end up sorting the two data files in different ways. If the data points aren't ordered then the filtering process goes haywire. I just added a two-line fix and pushed that to the repo - hopefully that will work for you?

**Comment by andreww** [comment]
Yes - that fix from yesterday (de9aae5fb40db9a73ae5e46de255add10da2222d) works perfectly for me and fixes this issue.
